deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -250,11 +272,15 @@ async def _get_jobs(con, token, job_ids):
 
 @app.post("/jobs")
 async def create_new_job(
-    request: Request, details: JobRequest, token=Depends(ProtectedAny(scope=["users:free"]))
+    request: Request,
+    details: JobRequest,
+    token=Depends(ProtectedAny(scope=PULSE3D_USER_SCOPES)),
 ):
     try:
         user_id = str(uuid.UUID(token["userid"]))
-        logger.info(f"Creating pulse3d job for upload {details.upload_id} with user ID: {user_id}")
+        customer_id = str(uuid.UUID(token["customer_id"]))
+        service = token["scope"][0].split(":")[0]
```

Comment:
It seems like with the new scopes we do a lot of splitting on ":", replacing etc. It would probably be good to make some utility functions that do this that we put in one of the core libs so we don't accidentally introduce a bug just in one place by splitting incorrectly or something else